src/components-examples/material/chips/chips-reactive-form/chips-reactive-form-example.html

```diff
@@ -0,0 +1,21 @@
+<section>
+    <h4>Chips inside of a Reactive form</h4>
+    <mat-form-field class="example-form-field">
+        <mat-label>Video keywords</mat-label>
+        <mat-chip-grid #reactiveChipGrid aria-label="Enter reactive form keywords" [formControl]="formControl">
```

Comment:
The indentation in this file seems off. Maybe it's using tabs instead of spaces?

Reply:
Updated to use spaces instead of tabs